hack/tools/create_kind.sh

```diff
@@ -8,7 +8,7 @@ op=$1
 
 source ./common.sh
 
-K8S_VERSION=${K8S_VERSION:-v1.16.9}
+K8S_VERSION=${K8S_VERSION:-v1.19.1}
```

Comment:
What in the changes require 1.19? It's rather new and we should probably peek an older version to test against (are you sure that these changes add something not supported in 1.16?)

Reply:
We upgraded `admissionReviewVersions` to `v1`. And when I run it with 1.16.15 from kind I'm getting an error in the webhook that it requires `v1beta1`. I'll try to make it compatible.